### PR TITLE
 [USING] WalletGeneration.md : Correct factual errors, and move some info to new/advanced password chapter

### PR DESCRIPTION
**Pull request by davterra**
OK, sorry to still be something of a github moron.

I tried to revert the inadvertent merge with one of my local branches, at the end of the predecessor to this PR, and screwed it up, so I just closed and opened a new one.

### CODE REVIEW

**Comment by yahiheb** [comment]
This PR replaces #593

> OK, sorry to still be something of a github moron.

No worries we are all learning here.

I think in the other PR there was no conflicts so the last commit (merge) was not necessary.

**Comment by davterra** [comment]
> This PR replaces #593
> 
> > OK, sorry to still be something of a github moron.
> 
> No worries we are all learning here.
> 
> I think in the other PR there was no conflicts so the last commit (merge) was not necessary.

That last merge was just me attempting to pull our revisions here back into my local branch. I'm clearly not good enough at Github to be drinking beer while I do stuff like this..... I tried to revert it after seeing what I had done, but I just made a bigger mess.

Thanks again.

File: docs/using-wasabi/README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Further tutorials about the different parts of the wallet, for newbies and power
 - [Cold-Wasabi Protocol](/using-wasabi/ColdWasabi.md)
 - [Headless Wasabi Daemon](/using-wasabi/Daemon.md)
 - [RPC Interface](/using-wasabi/RPC.md)
+- [Password Best Practices](/using-wasabi/PasswordBestPractices.md)
 - [Lurking Wife Mode](/using-wasabi/LurkingWifeMode.md)
```
Comment (MaxHillebrand) [inline docs/using-wasabi/README.md]:
```suggestion
```

File: docs/using-wasabi/README.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ Further tutorials about the different parts of the wallet, for newbies and power
 - [Cold-Wasabi Protocol](/using-wasabi/ColdWasabi.md)
 - [Headless Wasabi Daemon](/using-wasabi/Daemon.md)
 - [RPC Interface](/using-wasabi/RPC.md)
+- [Password Best Practices](/using-wasabi/PasswordBestPractices.md)
```
Comment (MaxHillebrand) [inline docs/using-wasabi/README.md]:
```suggestion
- [Lurking Wife Mode](/using-wasabi/LurkingWifeMode.md)
- [Password Best Practices](/using-wasabi/PasswordBestPractices.md)
```

**Comment by davterra** [comment]
> GitHell is fun, once you're in there you almost never come out again...
> "Have you tried closing and reopening the PR?" 
> 
> This is a very good change now @davterra - thank you kindly for sticking through with it!!

Thanks to each of you for your help.

**Pull request merged**